File: data/src/main/java/com/joxad/zikobot/data/module/spotify_api/model/SpotifyUser.java
package com.joxad.zikobot.data.module.spotify_api.model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * Created by josh on 11/03/16.
 */
@Data
@AllArgsConstructor(suppressConstructorProperties = true)
public class SpotifyUser  {

    @SerializedName("country")
    public String country;
    @SerializedName("display_name")
    public String displayName;
    @SerializedName("href")
    public String href;
    @SerializedName("id")
    public String id;
    @SerializedName("images")
    public List<Image> images = new ArrayList<Image>();
    @SerializedName("product")
    public String product;
    @SerializedName("type")
    public String type;
    @SerializedName("uri")
    public String uri;
}

File: data/src/main/java/com/joxad/zikobot/data/module/spotify_api/model/SpotifyUser.kt
package com.joxad.zikobot.data.module.spotify_api.model

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

/**
 * Created by josh on 11/03/16.
 */
class SpotifyUser {

    @SerializedName("country")
    var country: String? = null
    @SerializedName("display_name")
    var displayName: String? = null
    @SerializedName("href")
    var href: String? = null
    @SerializedName("id")
    var id: String? = null
    @SerializedName("images")
    var images: List<Image> = ArrayList()
    @SerializedName("product")
    var product: String? = null
    @SerializedName("type")
    var type: String? = null
    @SerializedName("uri")
    var uri: String? = null



}
